rotkehlchen/db/dbhandler.py

```diff
@@ -2502,10 +2502,18 @@ def query_owned_assets(self, cursor: 'DBCursor') -> List[Asset]:
                         if asset_id is not None:
                             results.add(Asset(asset_id).check_existence())
                     except UnknownAsset:
-                        self.msg_aggregator.add_warning(
-                            f'Unknown/unsupported asset {asset_id} found in the database. '
-                            f'If you believe this should be supported open an issue in github',
-                        )
+                        if table_name == 'manually_tracked_balances':
+                            self.msg_aggregator.add_warning(
+                                f'Unknown/unsupported asset {asset_id} found in the '
+                                f' manually tracked balances. Have you modified the assets DB? '
+                                f' Make sure that the aforementioned asset is in there.',
```

Comment:
This will have double space but we can change in other PR as it is really minor

Reply:
please find them out in the first review :vulcan_salute: